README.md

```diff
@@ -40,12 +40,19 @@ The table below shows all the pin connections between the nRF905 and the arduino
 ### Software
 The library can be imported by downloading the library as a .zip file and then add it to the Arduino IDE by using the library manager. 
 
+Alternatively, it is also possible to add the contents of the library into the same directory as your Arduino project
+
 ## Usage
 Examples are provided to give you a quick overview of how to use the library. 
 include the library into your project by adding the header file
-`#include <HomeyRadio.h>`.
 
-After that you need to create a Homey-Radio instance with a desired receiver address
+`#include <HomeyRadio.h>`
+
+or
+
+`#include "HomeyRadio.h"` if the library was added in the same directory as the project
```

Comment:
Ja ik ben het met je eens, ik zie nu idd hoe die library manager werkt. Maar misschien is het dan wel handig om nog even voor de herhaling uit te leggen welke stappen iemand moet nemen om een library te includen en wara hij hem dan kan vinden